uiowa/uiowa
# Article Contact reference field

Resolves #5999 




# How to test


`git checkout article-contacts && git pull && ddev blt ds --site=default && ddev drush @default.local uli`
Create an article or edit an existing article
Add existing Persons as a contact and/or create new contacts from the field
Check that the contact(s) show up on display

`ddev blt ds --site=now.uiowa.edu && ddev drush @now.local uli`
Create an article or edit an existing article
Add existing Persons as a contact and/or create new contacts from the field
Add existing Contacts as a contact and/or create new contacts from the field
Proofread the help text, check that it is clear and informative
Check that the contact(s) show up on display
Repeat for creating People and adding contact references to them

Check that features/splits that touch articles also work with the new contact (periodical, 91stmeridian.iwp, and iisc, but double check that there weren't others missed).

> Possibly reorganize fields on form display

No changes have been made in reorganizing the form display here; primarily, the addition of the new Point of Contact field has been added as collapsed by default in order to avoid mussing things up too much. Keep this in mind while testing, if there are other thoughts of ways to clean up/optimize the form display.
